app/controllers/agent/ChargeSummaryController.scala

```diff
@@ -50,7 +50,7 @@ class ChargeSummaryController @Inject()(chargeSummary: ChargeSummary,
     chargeSummary(
       documentDetailWithDueDate = documentDetailWithDueDate,
       implicitDateFormatter = dateFormatter,
-      paymentEnabled = isEnabled(Payment),
```

Comment:
Please remove this from the view instead of setting it to true

Reply:
done